Clinical-Genomics/cg
# adds new filter options to cli/status/cases

This PR adds new filter options to cli/status/cases

How to test:
0. install on stage on rasta
1. us
1. run cg status cases [options]

Expected outcome:
The cases should be listed/filtered the way you want to
